.github/workflows/test-js-packages.yml

```diff
@@ -104,3 +104,46 @@ jobs:
         # changes, and we don't want to enable that beahvior for _all_ our JS packages.
         run: |
           TURBO_API= turbo run check-types test --filter="!turborepo-repository" --filter={./packages/*}...[${{ github.event.pull_request.base.sha || 'HEAD^1' }}] --color --env-mode=strict
+
+  summary:
+    name: Turborepo JS Test Summary
```

Comment:
This needs to be added to the branch protection rules for main https://github.com/vercel/turbo/settings/branch_protection_rules/23353343